README.md

```diff
@@ -1,6 +1,5 @@
 # AWS Jupyter Proxy
 
-![CodeBuild](https://codebuild.us-west-2.amazonaws.com/badges?uuid=eyJlbmNyeXB0ZWREYXRhIjoiZGVqcG5MVEZZYkd2aFBBZnlSZ1R6d0s3U1JKR3pwOWR0UGJTdEp5bW9QWlVlMmdnTEJlenZUdVVkQjNzcVViMmlLQ1NGNS9yLzJEWkRpMzF5WUxnZTJVPSIsIml2UGFyYW1ldGVyU3BlYyI6Ikw5ck5kZk50ai9UU2pYanMiLCJtYXRlcmlhbFNldFNlcmlhbCI6MX0%3D&branch=master)
```

Comment:
Non blocking but it'd be nice to replace this badge with one from GitHub Actions https://docs.github.com/en/actions/managing-workflow-runs/adding-a-workflow-status-badge